internal/host/installcmd.go

```diff
@@ -136,6 +139,21 @@ func (i *installCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models
 	return []*models.Step{step}, nil
 }
 
+func getDiskFormatCmds(log logrus.FieldLogger, host models.Host) string {
+	var inventory models.Inventory
+	if err := json.Unmarshal([]byte(host.Inventory), &inventory); err != nil {
```

Comment:
Maybe we should fail? i don't like that we hide issues.

Reply:
Fixed